doc/user_guide/user_guide.md

```diff
@@ -466,6 +466,138 @@ CREATE OR REPLACE JAVA SET SCRIPT IMPORT_FILES(...) EMITS (...) AS
 The variable names may be different, please check out the Azure storage
 documentation.
 
+## Parallelism
+
+The setting for parallelism is **different** for import and export statements.
+
+### Import Parallelism Parameter
+
+In the import, the number of files in the storage path is distributed to the
+parallel running importer processes. These parallel processes can be controlled
+by setting the `PARALLELISM` parameter.
+
+By default, this parameter is set to the `nproc()`. The `nproc()` is an Exasol
+special SQL command that returns the total number of data nodes in your cluster.
+
+```sql
+IMPORT INTO <schema>.<table>
+FROM SCRIPT CLOUD_STORAGE_EXTENSION.IMPORT_PATH WITH
+  BUCKET_PATH     = 's3a://<S3_PATH>/*'
+  DATA_FORMAT     = 'ORC'
+  S3_ENDPOINT     = 's3.<REGION>.amazonaws.com'
+  CONNECTION_NAME = 'S3_CONNECTION'
+  PARALLELISM     = 'nproc()';
+```
+
+In the example above, `PARALLELISM` property value is set to `nproc()` which
+returns the number of physical data nodes in the cluster. Thus, the storage
+extension starts `nproc()` many parallel importer processes. The total number of
+files is distributed among these processes in a round-robin fashion and each
+process imports data from their own set of files.
+
+However, you can increase the parallelism by multiplying it with a number. For
+example, in order to start four times more processes, set it:
+
+```sql
+PARALLELISM = 'nproc()*4'
+```
+
+Or to a higher static number as `PARALLELISM = '16'` that will use 16 importer
+processes in total.
+
+We **recommend** to set the parallelism properly depending on the cluster
+resources (number of cores, memory per node), the number of files and the size
+of each file.
+
+### Export Parallelism Parameter
+
+In the export, the parallelism works differently compared to the import SQL
+statement.
+
+In the import statement, we are importing data from many files. Using the user
+provider parallelism number, we distribute these files into that many importer
+processes. For example, simply by taking modulo of file hash by parallelism
```

Comment:
Module? I understand that this might be a simplification for better understanding. But we already mentioned round robin above? Or there is no conflict?

Reply:
Good idea, I removed the sentence. It may confuse, and does not add any new information.